modules/mbedtls/Kconfig.psa

```diff
@@ -165,7 +165,7 @@ config PSA_HAS_HASH_SUPPORT
 config PSA_WANT_ALG_SHA_1
 	bool
 	prompt "PSA SHA1 support" if !PSA_PROMPTLESS
-	default y if !PSA_DEFAULT_OFF
```

Comment:
Is the correct Jira issue referenced?

I didn't find any more details there.

Reply:
Wrong jira in commit-message. NCSDK-17333 => NCSDK-17**8**33. Fixed!